ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -3015,6 +3015,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-zones-multisubnets-external-lb
+- as: vsphere-ipi-multi-vcenter-f28
+  cron: 4 4 12 * *
+  steps:
+    cluster_profile: vsphere-elastic
+    env:
+      FEATURE_GATES: '["ClusterAPIInstall=true","VSphereMultiVCenters=true"]'
```

Comment:
Sounds like `ClusterAPIInstall=true` is to enable capi, by default, vsphere is running installation with capi in 4.17 ?

Reply:
yes, i copy this config from e2e job. do you mean we should remove this feature gate setting?

Reply:
If capi is already the default behavior on 4.17 vsphere, then we do not need to set it here, only keep `VSphereMultiVCenters=true`